platforms/js/PixiWorkarounds.hx

```diff
@@ -672,7 +677,8 @@ class PixiWorkarounds {
 				{
 					let lineWidth;
 					lineWidth = widthContext.measureText(lines[i]).width / widthMulti;
-					lineWidth += (lines[i].length - 1) * style.letterSpacing + (style.wordSpacing ? style.wordSpacing * (lines[i].split(' ').length - 1) : 0.0);
+					lineWidth += (lines[i].length - 1) * style.letterSpacing / 2 + (style.wordSpacing ? style.wordSpacing * (lines[i].split(' ').length - 1) : 0.0);
```

Comment:
I'd say ((style.letterSpacing > 0) ? Math.floor(lines[i].length / 2) :  lines[i].length - 1) * style.letterSpacing is more accurate